tendrl/node_agent/message/handler.py

```diff
@@ -0,0 +1,53 @@
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+import gevent.event
+import gevent.greenlet
+from gevent.server import StreamServer
+from io import BlockingIOError
+import os
+from tendrl.commons.message import Message
+from tendrl.node_agent.message.logger import Logger
+
+
+RECEIVE_DATA_SIZE = 4096
+
+class MessageHandler(gevent.greenlet.Greenlet):
```

Comment:
If any issue occurs in this class, I feel it has to be written to stdout as the logger is not initialized yet and if not written to any place there is no way for user to know what went wrong.